internal/controller/ingress/tlsedge_controller.go

```diff
@@ -206,10 +224,20 @@ func (r *TLSEdgeReconciler) updateEdge(ctx context.Context, edge *ingressv1alpha
 }
 
 func (r *TLSEdgeReconciler) delete(ctx context.Context, edge *ingressv1alpha1.TLSEdge) error {
+	// Get all owned domains and delete them.
```

Comment:
The domains will cascade delete with the edge update? Or was more intended here to specifically handle domain deletion?

Reply:
I think that is an old comment I had back when I was thinking about deleting domains. I was playing around with adding owner references to domains to track what they were used by, but it didn't really pan out. Reserved domains are kind of tricky especially if it is an ngrok owned subdomain. If we delete it, then you may not be able to get it back. For Ingresses we don't actually delete them since that would allow someone else to snag it. I'll remove the comment there